test/unittests/util/test_format.py

```diff
@@ -186,6 +176,21 @@ def test_convert_scientific_notation(self):
                          "one point six seven two times ten to the power of "
                          "negative twenty seven")
 
+    def test_auto_scientific_notation(self):
+        self.assertEqual(
+            pronounce_number(1.1e-150), "one point one times ten to the "
+                                        "power of negative one hundred "
+                                        "and fifty")
+        self.assertEqual(
+            pronounce_number(sys.float_info.min), "two point two two times "
```

Comment:
I think this is a platform dependent value and should probably not be used in a test like this.